bindata/network/network-diagnostics/000-ns.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
extremely minor nit - this should just be in `bindata/network-diagnostics`, IMHO.

Reply:
done